tests/test_sparkpost_backend.py

```diff
@@ -49,7 +49,7 @@ class SparkPostBackendMockAPITestCase(RequestsBackendMockAPITestCase):
     def setUp(self):
         super().setUp()
         # Simple message useful for many tests
-        self.message = mail.EmailMultiAlternatives(
+        self.message = AnymailMessage(
```

Comment:
This change is no longer required (since you disabled mypy on tests).

But it raises a couple of questions: A lot of the tests use Anymail's added attributes directly on Django's EmailMessage or EmailMultiAlternatives, because this is a documented Anymail feature. (Anymail has always allowed duck typing for its added attributes.)

* Is there some special way we should be testing this works with mypy? (Other than just a test on EmailMessage or EmailMultiAlternatives that sets all of Anymail's attributes with type:ignore comments?)
* Is there some special way Anymail should be exposing this capability for library users? (Like an `AnymailMessageProtocol`?)